portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path

```diff
@@ -353,6 +353,16 @@
 	<td>//tr[contains(.,'${key_fieldName}')]//button[contains(.,'Apply')]</td>
 	<td></td>
 </tr>
+<tr>
```

Comment:
@Jeremy-Ch can you please use `Button#ANY` and `Button#ANY_DISABLED` instead of creating these?

Reply:
I have tried these paths, they can not be used for the field translation button.